BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -4004,13 +4002,15 @@ void TakeSample()
             ++passNumber;
             if ((passNumber %= (int)AI.autoTuningNumSamples) == 0)
             {
-                if (currentField == "base") // Update the UI with the current loss values at the base point.
-                {
-                    AI.autoTuningLossLabel = $"{lossSamples[currentField].Average():F4} @ {headingChange:F0}°";//$"Fast: {fastResponseLoss:F4}, Osc: {oscillationLoss:F4}";
-                }
                 ++currentFieldIndex;
                 UpdatePIDValues((currentFieldIndex %= fieldNames.Count) == 0);
             }
+            
+            // Change heading for next sample
+            headingChange = Mathf.Sign(headingChange) * (30f + passNumber * (90f / (AI.autoTuningNumSamples - 1f)));
```

Comment:
This should probably be `30f + (passNumber + 0.5f) * 90f / AI.autoTuningNumSamples` to approximate the integral `∫f(x, θ)dθ` as the sum of the bars centred at the sample points (e.g., https://www.researchgate.net/figure/Approximation-to-the-area-under-a-curve-using-the-midpoint-rule-Derived-from-a-figure_fig2_288833365). The initial value in `ResetSamples` would also need to be adjusted by half a step too.

Reply:
I've made this change in the merged PR.